mypy/semanal.py

```diff
@@ -947,7 +947,10 @@ def prepare_method_signature(self, func: FuncDef, info: TypeInfo, has_self_type:
             if func.name in ["__init_subclass__", "__class_getitem__"]:
                 func.is_class = True
             if not func.arguments:
-                self.fail("Method must have at least one argument", func)
+                self.fail(
+                    "Method must have at least one argument. Did you forget the self argument?",
```

Comment:
```suggestion
                    'Method must have at least one argument. Did you forget the "self" argument?',
```
I believe our convention is to put code samples in quotes in error messages.